gapic/src/main/com/google/gapid/widgets/Widgets.java

```diff
@@ -788,4 +788,8 @@ public static void disposeAllChildren(Composite parent) {
       child.dispose();
     }
   }
+
+  public void setClipboardContent(String s) {
```

Comment:
We haven't really been adding forwarders like this in either the `Models` nor the `Widgets` class, so it seem a bit odd to add just this one now. I'd say, simply call the `CopyPaste.setContent` method directly via `widgets.copypaste.setContents`.